pyaedt/rpc/rpyc_services.py

```diff
@@ -823,6 +825,18 @@ def aedt_grpc(port=None, beta_options=None, use_aedt_relative_path=False, non_gr
             command = [aedt_exe, "-grpcsrv", str(port), ng_feature]
 
         subprocess.Popen(command)
+        timeout = 60
+        s = socket.socket()
+        machine_name = socket.getfqdn()
+        while timeout > 0:
+            try:
+                s.connect((machine_name, port))
+            except socket.error:
+                timeout -= 2
+                time.sleep(2)
+            else:
+                s.close()
```

Comment:
@maxcapodi78 I am assuming here we are always closing the socket in the `else` block if no exception is raised because it is just to check the gRPC port.

Reply:
LGTM.

Reply:
correct